docs/source/deployment/databricks/databricks_notebooks_development_workflow.md

```diff
@@ -0,0 +1,293 @@
+    recurse=True,
+)
+```
+
+Run this cell to copy the complete directory and its contents from your Repo to DBFS.
+
+To ensure that your data was copied correctly, you can list the contents of the destination directory in DBFS. Create a new cell underneath the first cell and enter the following code:
+
+```python
+dbutils.fs.ls("dbfs:/FileStore/iris-databricks/data")
+```
+
+Run this command to displays the contents of your project's `data/` directory. You can expect to see the following structure:
+
+```bash
+[FileInfo(path='dbfs:/FileStore/iris-databricks/data/01_raw', name='01_raw', size=...),
+ FileInfo(path='dbfs:/FileStore/iris-databricks/data/02_intermediate', name='02_intermediate', size=...),
+ FileInfo(path='dbfs:/FileStore/iris-databricks/data/03_primary', name='03_primary', size=...),
+ FileInfo(path='dbfs:/FileStore/iris-databricks/data/04_feature', name='04_feature', size=...),
+ FileInfo(path='dbfs:/FileStore/iris-databricks/data/05_model_input', name='05_model_input', size=...),
+ FileInfo(path='dbfs:/FileStore/iris-databricks/data/06_models', name='06_models', size=...),
+ FileInfo(path='dbfs:/FileStore/iris-databricks/data/07_model_output', name='07_model_output', size=...),
+ FileInfo(path='dbfs:/FileStore/iris-databricks/data/08_reporting', name='08_reporting', size=...)]
+```
+
+After these cells have successfully run, you should comment the code inside them so their operations are not unnecessarily performed during notebook runs. The cells should appear as below:
+
+**Cell 1:**
+
+```ipython
+#dbutils.fs.cp(
+#    "file:///Workspace/Repos/<databricks_username>/iris-databricks/data",
+#    "dbfs:/FileStore/iris-databricks/data",
+#    recurse=True,
+#)
+```
+
+**Cell 2:**
+
+```ipython
+#dbutils.fs.ls("dbfs:/FileStore/iris-databricks/data")
+```
+
+### Run your project
+
+Create **four new cells** inside your notebook. You will fill these cells with code that runs your project. When copying the following code snippets, remember to replace `<databricks_username>` with your username on Databricks such that `project_root` correctly points to your project's location.
+
+1. Before you import and run your Python code, you'll need to install your project's dependencies on the cluster attached to your notebook. Your project has a `requirements.txt` file for this purpose. Add the following code to the first new cell to install the dependencies:
+
+```ipython
+%pip install -r "/Workspace/Repos/<databricks_username>/iris-databricks/src/requirements.txt"
+```
+
+2. To run your project in your notebook, you must load the Kedro IPython extension. Add the following code to the second new cell to load the IPython extension:
+
+```ipython
+%load_ext kedro.ipython
+```
+
+3. Loading the extension allows you to use the `%reload_kedro` line magic to load your Kedro project. Add the following code to the third new cell to load your Kedro project:
+
+```ipython
+%reload_kedro /Workspace/Repos/<databricks_username>/iris-databricks
```

Comment:
Probably

```suggestion
%reload_kedro /Workspace/Repos/<databricks_username>/iris-databricks --env=databricks
```

Reply:
There's no databricks environment in the `databricks-iris` starter, see comment above.